hurain/Tic_tac_toe_del3.cpp

```diff
@@ -0,0 +1,185 @@
+#include <iostream>
+#include <limits>
+#include <cstdlib>
+#include <ctime>
+#include <ncurses.h>
+
+using namespace std;
+
+class Board {
+public:
+    char spaces[9];
```

Comment:
using the array to 9 characters simplify the solution compare to an array of 3x3

Reply:
Hey Chi Thu, what do you want me to do here?